### DISCUSSION TITLE
Aggregate of 30 day installs is greater than 90 days

### DISCUSSION THREAD

**Discussion by bkowshik** [Homebrew/discussions#4548] [Everyday usage]
### Output of `brew config`

```shell
...
```

### Output of `brew doctor`

```shell
...
```

### Description of issue

The webpage for every package has an Analytics section at the bottom that shows:
1. Installs (30 days)
2. Installs (90 days)
3. Installs (365 days)

Don't understand why/how numbers for `90 day` is lower in comparison to the `30 day` numbers.
- https://formulae.brew.sh/cask/quarto
- https://formulae.brew.sh/cask/firefox

<img width="1278" alt="Screenshot 2023-06-01 at 05 56 04" src="https://github.com/Homebrew/discussions/assets/2899501/1f9d6082-92ba-4123-82a8-a880428272da">

**Comment by SMillerDev** [comment]
Because analytics to Google Analytics doesn't seem to work well, and 30 days comes from influxdb.

**Reply by carlocab** [reply]
We should probably just stop publishing and collecting Google Analytics; it's clearly very broken.

**Reply by SMillerDev** [reply]
Yeah, that's the plan once we have the InfluxDB data tweaked to our liking. I much prefer to spend my time on that right now.
